src/sharedInterfaces/connectionGroup.ts

```diff
@@ -42,3 +43,7 @@ export interface ConnectionGroupSpec {
     description?: string;
     color?: string;
 }
+
+export interface ConnectionGroupConnectionProfile extends IConnectionInfo {

```

Comment:
I would get rid of this, and just have `DockerConnectionProfile` extend `IConnectionDialogProfile` directly